js/react_native/android/build.gradle

```diff
@@ -20,6 +20,24 @@ def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['OnnxruntimeModule_' + name]).toInteger()
 }
 
+def checkIfOrtExtensionsEnabled() {
+  // locate user's project dir
+  def reactnativeRootDir = project.rootDir.parentFile
+  // get package.json file in root directory
+  def packageJsonFile = new File(reactnativeRootDir, 'package.json')
+  // read field 'onnxruntimeExtensionsEnabled'
+  if (packageJsonFile.exists()) {
+    def packageJsonContents = packageJsonFile.getText()
+    def packageJson = new groovy.json.JsonSlurper().parseText(packageJsonContents)
+    return packageJson.onnxruntimeExtensionsEnabled == 'true'
```

Comment:
> onnxruntimeExtensionsEnabled

updated to `onnxruntimeExtensionsEnabled` as discussed as `ort` is opaque in the context. @skottmckay 

will update the corresponding field in sample usage package.json file as well